etc/inc/config.lib.inc

```diff
@@ -919,6 +919,20 @@ function get_config_backup_count() {
 function pfSense_clear_globals() {
 	global $config, $FilterIfList, $GatewaysList, $filterdns, $aliases, $aliastable;
 
+	$error = error_get_last();
+	if( $error !== NULL) {
+		$errtype = $error["type"];
+		if ($errtype != E_NOTICE) {
+			// there are a lot of 'innocent' errors.. so skip reporting E_NOTICE
+			$errfile = $error["file"];
+			$errline = $error["line"];
+			$errstr  = $error["message"];
+			$str = "PHP ERROR Type:$errtype, File:$errfile, Line: $errline, Message: $errstr";
```

Comment:
These variables are unecessary ($errfile, $errline and $errstr), you can print $error["file"], $error["line"] and $error["message"] direct when setting $str, just enclose them in {}.
```
$str = "PHP ERROR Type:{$error['type']}, File:{$error['file']}, Line: {$error['line']}, Message: {$error['message']}";
```